components/suggestion/switch_channel_provider.test.jsx

```diff
@@ -77,9 +77,9 @@ describe('components/SwitchChannelProvider', () => {
     };
 
     const switchProvider = new SwitchChannelProvider();
-    const mockStore = configureStore();
+    let mockStore = configureStore();
```

Comment:
Perhaps instead of initializing `mockStore` for the whole test suite and the overriding it in the new test case, maybe we should just initialize individual versions of `mockStore` in each test case instead? As it is, I think any test cases added after your new one will also pick up the modified state which is a little weird.

Reply:
Yeah, I thought about that and would prefer that actually. I didn't know what is the usual convention followed here. It would have just created a lot more duplicated code. Will change it. Thanks.

Reply:
I don't think it should be too much duplicated code if we share the base state. I think calling `const mockStore = configureStore;` in each test case is important to prevent tests from interfering with others' testing conditions though. I'm also a fan of the Go idea of "a little duplication is better than a little dependency" though.